src/main.py

```diff
@@ -248,13 +255,13 @@ def page_not_found(e):
 
 
 @app.errorhandler(500)
-def server_error(e):
+def server_error_500(e):
```

Comment:
While we're at it we can change the function name to be more descriptive.
```suggestion
def handle_internal_server_error(e):
```